core/src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -326,23 +326,23 @@ public void onEnable() {
         this.liteCommands = LiteBukkitAdventurePlatformFactory.builder(server, "eternalcore", false, this.audiencesProvider, this.miniMessage)
 
             // Arguments (include optional)
-            .argument(String.class, "player",   new StringNicknameArgument(server))
-            .argument(GameMode.class,               new GameModeArgument(this.viewerProvider, this.translationManager, commandConfiguration.argument))
-            .argument(NoticeType.class,             new NoticeTypeArgument(this.viewerProvider, this.translationManager))
-            .argument(Warp.class,                   new WarpArgument(this.warpManager, this.translationManager, this.viewerProvider))
-            .argument(Enchantment.class,            new EnchantmentArgument(this.viewerProvider, this.translationManager))
-            .argument(User.class,                   new UserArgument(this.viewerProvider, this.translationManager, server, this.userManager))
-            .argument(Player.class,                 new PlayerArgument(this.viewerProvider, this.translationManager, server))
-            .argument(Player.class, "request",  new RequesterArgument(this.teleportRequestService, this.translationManager, this.viewerProvider, server))
-            .argument(Duration.class, "duration", new DurationArgument(this.viewerProvider, this.translationManager))
-            .argument(Integer.class, "speed",    new SpeedArgument(this.viewerProvider, this.translationManager))
+            .argument(String.class, StringNicknameArgument.KEY, new StringNicknameArgument(server))
+            .argument(GameMode.class,                           new GameModeArgument(this.viewerProvider, this.translationManager, commandConfiguration.argument))
+            .argument(NoticeType.class,                         new NoticeTypeArgument(this.viewerProvider, this.translationManager))
+            .argument(Warp.class,                               new WarpArgument(this.warpManager, this.translationManager, this.viewerProvider))
+            .argument(Enchantment.class,                        new EnchantmentArgument(this.viewerProvider, this.translationManager))
+            .argument(User.class,                               new UserArgument(this.viewerProvider, this.translationManager, server, this.userManager))
+            .argument(Player.class,                             new PlayerArgument(this.viewerProvider, this.translationManager, server))
+            .argument(Player.class, RequesterArgument.KEY,      new RequesterArgument(this.teleportRequestService, this.translationManager, this.viewerProvider, server))
+            .argument(Duration.class, DurationArgument.KEY,     new DurationArgument(this.viewerProvider, this.translationManager))
+            .argument(Integer.class, SpeedArgument.KEY,         new SpeedArgument(this.viewerProvider, this.translationManager))
```

Comment:
Moim zdaniem te wcięcia są do usunięcia. W rzeczywistości dodaje to bardzo mało readability, nie jest to jakaś kolosalna zmiana.

Reply:
Już jakiś czas temu rozmawialiśmy o tym że tak nie powinno to wyglądać bo nie jest to czytelne. 

![image](https://user-images.githubusercontent.com/77621271/222525370-c2b45d5c-4a63-4525-97be-3b26e7cc9422.png)